src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_help.py

```diff
@@ -72,17 +72,83 @@
     type: group
     short-summary: Generic commands to manage Azure resources
 """
+helps['resource list'] = """
+    type: command
+    short-summary: list resource
+    examples:
+        - name: list all resource in a region
```

Comment:
I know this is not related to resource Id change, but "list all resource in a region" ? Why? Does it list all resources in a subscription?